client/executor/common/src/error.rs

```diff
@@ -160,3 +163,38 @@ pub enum WasmError {
 	#[error("{0}")]
 	Other(String),
 }
+
+/// An error message with an attached backtrace.
+#[derive(Debug)]
+pub struct MessageWithBacktrace {
+	/// The error message.
+	pub message: String,
+
+	/// The backtrace associated with the error message.
+	pub backtrace: Option<Backtrace>,
```

Comment:
Does this mean it might exist? I'd call it 'maybe_backtrace' then.

Reply:
Hm... I *think* idiomatically people don't usually prepend `maybe_` on `Option` fields in Rust since the type is there anyway to signify that it's optional? At least I don't see it done often. And ideally we'd like this to be not an `Option`; it's only an `Option` because `wasmi` doesn't currently support backtraces.

But I don't really care *either* way, so I'm fine with renaming it if you want.